roles/common/tasks/system-file-permissions.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Set permissions for system log directory recurisively
+  file: path=/var/log state=directory recurse=yes mode=0640
```

Comment:
On subsequent runs this will change ownership of some of the subdirs that we explicitly set to have different ownerships.